contrib/vagrant/scripts/04-install-kubectl.sh

```diff
@@ -16,6 +16,22 @@ log "Installing kubernetes kubectl..."
 
 set -e
 
+# Add aliases and bash completion for kubectl
+cat <<EOF >> /home/vagrant/.bashrc
+# kubectl
+source <(kubectl completion bash)
+alias k='kubectl'
+complete -F __start_kubectl k
+alias ks='kubectl -n kube-system'
+complete -F __start_kubectl ks
+alias wk='watch -n2 kubectl get pods'
```

Comment:
Any reason not to put `-o wide` in here by default? You already mentioned that you want that in the commit message.

Reply:
I didn't include it initially because `wk -o wide --show-labels` doesn't fit my screen (and I expect other's). But I rarely use `--show-labels` with `-o wide` so let's include it.